### PR TITLE
Fix all links in content\workers\*.md

### PR DESCRIPTION
**Pull request by firtina-improbable**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

Also fixes a heading for "Checking Out" in the glossary

### CODE REVIEW

**Comment by firtina-improbable** [comment]
WIP, needs one more commit for syntax fixing

**Comment by firtina-improbable** [comment]
Ready for review @jessicafalk @samc-improbable

File: docs/content/workers/api-worker.md [hunk 1]
```diff
@@ -2,33 +2,33 @@
...
 
-Upon successfully connecting to the SpatialOS Runtime and creating your worker and its corresponding [ECS world](../glossary.md#unity-ecs-world), the following systems are added by the `Worker` class your [worker’s world](../glossary.md#workers-world). These systems  ensure that any changes in any SpatialOS entity is correctly synchronized between the SpatialOS [Runtime](../glossary.md#spatialos-runtime) and the [worker's view](../glossary.md#workers-view).
+Upon successfully connecting to the SpatialOS Runtime and creating your worker and its corresponding [ECS world]({{urlRoot}}/content/glossary.md#unity-ecs-world), the following systems are added by the `Worker` class your [worker’s world]({{urlRoot}}/content/glossary.md#worker-s-world). These systems  ensure that any changes in any SpatialOS entity is correctly synchronized between the SpatialOS [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime) and the [worker's view]({{urlRoot}}/content/glossary.md#worker-s-view).
 
 ## Systems ensuring synchronization between the Runtime and workers
```
Comment (jessicafalk) [inline docs/content/workers/api-worker.md]: uhh didn't see that before. will have to go in like this for now, but not all of them are strictly speaking responsible for synchronization

File: docs/content/workers/api-worker.md [hunk 2]
```diff
@@ -82,42 +70,38 @@ Callback parameters:
...
 ```csharp
-
 static async Task<Worker> CreateWorkerAsync(ReceptionistConfig config, ILogDispatcher logger, Vector3 origin);
-
 ```
 
 Parameters:
 
-  * `ReceptionistConfig config`: The connection configuration used to connect via the [Receptionist](../connecting-to-spos.md).
+  * `ReceptionistConfig config`: The connection configuration used to connect via the [Receptionist]({{urlRoot}}/content/connecting-to-spos.md).
```
Comment (jessicafalk) [inline docs/content/workers/api-worker.md]: that link should go to this heading: Receptionist service connection flow

File: docs/content/workers/api-worker.md [hunk 3]
```diff
@@ -82,42 +70,38 @@ Callback parameters:
...
 ```csharp
-
 static async Task<Worker> CreateWorkerAsync(LocatorConfig config, Func<DeploymentList, string> deploymentListCallback, ILogDispatcher logger, Vector3 origin);
-
 ```
 
 Parameters:
 
-* `LocatorConfig config`: The connection configuration used to connect to the [Runtime](../glossary.md#spatialos-runtime) via the [Locator](../connecting-to-spos.md) flow.
+* `LocatorConfig config`: The connection configuration used to connect to the [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime) via the [Locator]({{urlRoot}}/content/connecting-to-spos.md) flow.
```
Comment (jessicafalk) [inline docs/content/workers/api-worker.md]: should go to that heading: Locator connection flow

File: docs/content/workers/worker-entity.md [hunk 4]
```diff
@@ -1,20 +1,19 @@
...
- 	* `OnConnected`: the worker just connected to the SpatialOS [Runtime](../glossary.md#spatialos-runtime).
- 	* `OnDisconnected`: the worker just disconnected from the SpatialOS [Runtime](../glossary.md#spatialos-runtime). This is an `ISharedComponentData` and stores the reason for the disconnection as a `string`.
+  * send and receive [commands (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#command) before the worker has checked out any SpatialOS entities.
+  * register changes to the state of the Runtime connection (that is whether the worker is connected to the [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime) or not) by filtering for the following [temporary components]({{urlRoot}}/content/ecs/temporary-components.md):
+     * `OnConnected`: the worker just connected to the SpatialOS [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime).
+     * `OnDisconnected`: the worker just disconnected from the SpatialOS [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime). This is an `ISharedComponentData` and stores the reason for the disconnection as a `string`.
```
Comment (jessicafalk) [inline docs/content/workers/worker-entity.md]: linking multiple times for the same term, but should be fine for the first version

File: docs/content/workers/workers-in-the-gdk.md [hunk 5]
```diff
@@ -2,54 +2,55 @@
 # Workers: Workers in the GDK
 _This document relates to both GameObject-MonoBehaviour and ECS workflows._
 
-Before reading this document, make sure you are familiar with the [GameObject-MonoBehaviour and  ECS workflow](../intro-workflows-spos-entities.md).
+Before reading this document, make sure you are familiar with the [GameObject-MonoBehaviour and  ECS workflow]({{urlRoot}}/content/intro-workflows-spos-entities.md).
 
 ## What is a SpatialOS worker?
 
-The SpatialOS [Runtime](../glossary.md#spatialos-runtime) manages your [game world](../glossary.md#spatialos-game-world) by keeping track of all [SpatialOS entities](../glossary.md#spatialos-entity) and the current state of their [components](../glossary.md#spatialos-componants).
+The SpatialOS [Runtime]({{urlRoot}}/content/glossary.md#spatialos-runtime) manages your [game world]({{urlRoot}}/content/glossary.md#world) by keeping track of all [SpatialOS entities]({{urlRoot}}/content/glossary.md#spatialos-entity) and the current state of their [components]({{urlRoot}}/content/glossary.md#spatialos-component).
```
Comment (jessicafalk) [inline docs/content/workers/workers-in-the-gdk.md]: #world or #spatialos-world?

File: docs/content/workers/workers-in-the-gdk.md [hunk 6]
```diff
@@ -2,54 +2,55 @@
...
+  * [API - worker]({{urlRoot}}/content/workers/api-worker.md)
 
-  * GameObject-MonoBehaviour workflow - [Workers: Creating workers with WorkerConnector](../gameobject/gomb-creating-workers-with-workerconnector.md)
+  * GameObject-MonoBehaviour workflow - [Workers: Creating workers with WorkerConnector]({{urlRoot}}/content/gameobject/gomb-creating-workers-with-workerconnector.md)
 
-  * ECS workflow - [API - workers system](./api-workers-system.md)
-  <br/>(Workers: Accessing information about the worker at Runtime)
+  * ECS workflow - [API - workers system]({{urlRoot}}/content/workers/api-worker-system.md)
+  
+  * ECS workflow - [ECS: Accessing information about the worker during runtime]({{urlRoot}}/content/ecs/accessing-worker-info.md)
```
Comment (jessicafalk) [inline docs/content/workers/workers-in-the-gdk.md]: that doc will be removed. use `workers/api-worker-system` instead

Comment (jessicafalk) [reply docs/content/workers/workers-in-the-gdk.md]: and it shouldn't say ECS

File: docs/content/workers/workers-in-the-gdk.md (hunk 5 above)
Comment (firtina-improbable) [reply docs/content/workers/workers-in-the-gdk.md]: o, looks like they are `#suspish`

**Pull request merged**